stdlib/_interpqueues.pyi

```diff
@@ -0,0 +1,16 @@
+from typing import SupportsIndex
+
+class QueueError(RuntimeError): ...
+class QueueNotFoundError(QueueError): ...
+
+def bind(qid: SupportsIndex) -> None: ...
+def create(maxsize: SupportsIndex, fmt: SupportsIndex) -> int: ...
+def destroy(qid: SupportsIndex) -> None: ...
+def get(qid: SupportsIndex) -> tuple[object, int]: ...
+def get_count(qid: SupportsIndex) -> int: ...
+def get_maxsize(qid: SupportsIndex) -> int: ...
+def get_queue_defaults(qid: SupportsIndex) -> tuple[int]: ...
+def is_full(qid: SupportsIndex) -> bool: ...
+def list_all() -> list[tuple[int, int]]: ...
+def put(qid: SupportsIndex, obj: object, fmt: SupportsIndex) -> None: ...
```

Comment:
If I understand this correctly (which I might not), the allowed values for `obj` depend on `fmt`. This would mean either using overloads or – easiest for now – `Any` for `obj`.

Reply:
Does using `Any` versus `object` really serve much of an advantage for param like this?

Reply:
This is mainly a documentation thing: "We can't accurately type this argument" instead of "this argument accepts all possible values" (like for example `str()` does).

Reply:
Ah I see, makes sense.

Reply:
Let's keep things simple for now that in this case, and we can see about adding the overloads in a follow-up.

Reply:
https://github.com/python/typeshed/pull/12240/commits/98a79ac82df0ffb99eef344f2d4dceacba533b0f